CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+
+## 0.1.0 (2020-02-11)
+
+* Improve documentation of the README and create CONTRIBUTING document
+  for developers (#2 and #3 by @legionth)
+* Add composer.json to identify dependencies and adding the repository
+  to packagist.org (#1 and #2 by @legionth)
```

Comment:
I think I would prefer it, if the #'s and @'s were actual links. They are not displayed as such if I look at the rendered file on GitHub.

Also shouldn't it be #4 instead of #2 for the second item?

Reply:
The `#` PRs and `@` are internal identifiers by GitHub. We could use general links, but this would be a lot of work each for a changelog. I prefer this way to keep it nice and easy :)